target/linux/mediatek/filogic/base-files/etc/hotplug.d/firmware/11-mt76-caldata

```diff
@@ -57,6 +56,10 @@ case "$FIRMWARE" in
 	glinet,gl-mt6000)
 		caldata_extract_mmc "factory" 0x0 0x1000
 		;;
+	asus,tuf-ax6000)
```

Comment:
Entries in this file should be in alphabetical order. This should be added above the glinet one.